project/plugins.sbt

```diff
@@ -5,6 +5,6 @@ resolvers += "Typesafe Releases" at "https://repo.typesafe.com/typesafe/releases
 
 addSbtPlugin("uk.gov.hmrc"        %  "sbt-auto-build"         % "3.0.0")
```

Comment:
auto build is now 3.3.0

Can you also try to update the test libraries. The sclatestplus play is usually good to use 5.0.0